components/swap/SurchargeCard.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div class="nft-card bg-k-grey-light !px-4 !py-6 flex flex-col items-center justify-between gap-4">
+    <div class="text-lg text-center font-bold capitalize">
+      {{ $t('swap.surchargeTitle') }}
+    </div>
+
+    <div class="flex flex-col gap-1 items-center justify-center rounded-2xl bg-white w-full h-full">
+      <div class="text-[31px]">
+        {{ formatted }}
+      </div>
```

Comment:
can't see token amount in dark mode

<img src=https://github.com/user-attachments/assets/05d2de6d-6367-4f63-b50f-f80495d11199 width=400 />

Reply:
updated 
<img width="309" alt="image" src="https://github.com/user-attachments/assets/8d2f79bd-fabf-48da-880b-27ea2cf3c02b" />